env.example

```diff
@@ -31,7 +31,6 @@ REQ_SERVICE_ID=req_id_dev
 MIGRATE_SERVICES_PREFERENCES_PROFILE_QUEUE_NAME=profile-migrate-services-preferences-from-legacy
 
 EUCOVIDCERT_PROFILE_CREATED_QUEUE_NAME=eucovidcert-profile-created
```

Comment:
this can also be removed from the `env.example`. probably it was missed time ago.

Reply:
Solved in 0d5f0a4